lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -65,74 +65,76 @@
 
 
 def load_config(config_data):
-    if 'aws' not in config_data:
-        raise Exception("'aws' section is mandatory in the configuration")
+    if "aws" not in config_data:
+        config_data["aws"] = {}
 
-    if not {'access_key_id', 'secret_access_key'}.issubset(set(config_data['aws'])):
-        raise Exception("'access_key_id' and 'secret_access_key' are mandatory under the 'aws' section of the configuration")
+    if "secret_access_key" in config_data["aws"] or "access_key_id" in config_data["aws"]:
+        logger.warning("Using 'secret_access_key' and 'access_key_id' in lithops configuration is not recommended "
+                       "- Use boto3 configuration file in ~/.aws or environment variables instead "
+                       "(https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html)")
```

Comment:
No need for any warning for now. We still have to decide whether we want to deprecate it or not